### PR TITLE
ISPN-16331 Cache alias tutorial

### PR DESCRIPTION
**Pull request by karesti**
https://issues.redhat.com/browse/ISPN-16331

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -14,6 +14,25 @@
     <properties>
         <testcontainers.version>1.19.1</testcontainers.version>
     </properties>
+    <profiles>
```
Comment (pruivo) [inline pom.xml:17]: you don't need a profile for this. You can add `<repositories>` tag directly.

File: infinispan-remote/cache-alias/pom.xml [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>infinispan-simple-tutorials-remote-cache-alias</artifactId>
+    <parent>
+        <relativePath>../../pom.xml</relativePath>
+        <version>1.0.0-SNAPSHOT</version>
+        <groupId>org.infinispan.tutorial.simple</groupId>
+        <artifactId>infinispan-simple-tutorials</artifactId>
+    </parent>
+    <name>Infinispan Simple Tutorials: Cache Alias</name>
```
Comment (pruivo) [inline infinispan-remote/cache-alias/pom.xml:11]: niptpicking, to be consistent with the name in embedded tutorial
```suggestion
    <name>Infinispan Simple Tutorials: Remote Cache Alias</name>
```

**Comment by karesti** [comment]
@pruivo I reported the changes, thanks!

File: pom.xml [hunk 3]
```diff
@@ -14,6 +14,18 @@
     <properties>
         <testcontainers.version>1.19.1</testcontainers.version>
     </properties>
+    <repositories>
+        <repository>
+            <id>infinispanSnapshots</id>
+            <releases>
+                <enabled>true</enabled>
```
Comment (pruivo) [inline pom.xml:21]: AFAIK, `https://s01.oss.sonatype.org/content/repositories/snapshots` does not expose releases (Final, CRs, Dev...)
```suggestion
                <enabled>false</enabled>
```

**Comment by pruivo** [comment]
@karesti we need to branch for 15.0.x before I can merge it
@pminz the product scripts need to be updated to use the `15.0.x` branch 
Thanks!

**Comment by pruivo** [comment]
@karesti I'm confused. Isn't the cache alias a 15.1 feature? We use the main branch to build the product tutorials, would the cache alias work with 15.0?

**Comment by karesti** [comment]
@pruivo I'm targeting "development' branch in this PR and using snapshots
When a 15.1 will be released, we need to move main to 15.0.x and main will be 15.1. development will always be snapshot infinispan. 15.0 tutorials will remain unchanged, unless someone raises an issue

**Comment by pruivo** [comment]
@karesti My bad! I didn't notice it was targeting the `development` branch. 
Merged! Thanks!

**Pull request merged**